server/server.coffee

```diff
@@ -5,12 +5,41 @@
 # Use Bluebird promises instead of native ones
 global.Promise = require 'bluebird'
 _ = require '../lib/utils'
+CONFIG = require 'config'
+breq = require 'bluereq'
+{ host:elasticHost, indexes } = CONFIG.elastic
 
-app = require('express')()
-bodyParser = require 'body-parser'
-app.use bodyParser.json()
-app.use bodyParser.urlencoded({ extended: true })
+ensureElasticIndex = ->
+  breq.get elasticHost
+  .then createIndex
+  .catch waitForElastic
 
-app.post '/', require('./post')
+createIndex = ->
+  unless indexes.inventaire
+    return console.log "no inventaire indexes"
 
-app.listen port, -> _.info "server listening on port #{port}"
+  breq.get "#{elasticHost}/#{indexes.inventaire}"
+  .catch (err)->
+    breq.put "#{elasticHost}/#{indexes.inventaire}"
+    .catch (err)->
+      _.error err "invalid elastic index name"
```

Comment:
you can't set a label that says what the error is as it might be something different like a connection error, you can only help locate the error with a label like `"failed to create #{elasticHost}/#{indexes.inventaire}"`

Reply:
also, a comma is missing after `err`